packages/web/src/components/PageBanner.tsx

```diff
@@ -2,6 +2,9 @@ import BackButton from "./BackButton";
 import BannerColorContext from "./BannerColorContext";
 import Box from "@material-ui/core/Box";
 import Grid from "@material-ui/core/Grid";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
```

Comment:
```suggestion
import List from "@material-ui/core/List";
import ListItem from "@material-ui/core/ListItem";
import ListItemIcon from "@material-ui/core/ListItemIcon";
import ListItemText from "@material-ui/core/ListItemText";
```
I think we're going to need the `ListItemText` component as well to match the [MUI List Examples](https://v4.mui.com/components/lists/).